subsys/net/lib/lwm2m/lwm2m_rd_client.c

```diff
@@ -127,6 +127,25 @@ struct lwm2m_rd_client_info {
  */
 static char query_buffer[MAX(32, sizeof("ep=") + CLIENT_EP_LEN)];
 
+static struct lwm2m_message *rd_get_message(void)
+{
+	if (client.rd_message.ctx) {
+		return NULL;
+	}
+
+	client.rd_message.ctx = client.ctx;
+	return &client.rd_message;
+
+}
+
+struct lwm2m_message *lwm2m_get_rd_message(void)
```

Comment:
`rd_get` and `get_rd` are way too close together. It is not clear which does what.
Please refactor either of these.

For example this function could be
```suggestion
struct lwm2m_message *lwm2m_get_ongoing_rd_msg(void)
```

Reply:
Ok Thanks for good proposal